### PR TITLE
Replace docker-registry-*.json with osadm registry

### LINKED ISSUE #1964: Remove sample-app's docker-registry-{config,template}.json
Remove sample-app's docker-registry-{config,template}.json since we have `osadm registry` now. Update the docs and tests that use these files accordingly as well.

### PR DESCRIPTION
**Pull request by ncdc**
Remove sample-app's docker-registry-{config,template}.json since we have
osadm registry now. Update the docs and tests that use these files
accordingly as well.

Fixes #1964

### CODE REVIEW

**Comment by ncdc** [comment]
@smarterclayton @bparees PTAL

**Comment by smarterclayton** [comment]
LGTM but ben gets final say.

File: docs/debugging-openshift.md [hunk 1]
```diff
@@ -53,13 +53,19 @@ Docker Registry
...
+
+If it's not running, you will instead see:
 
-    $ osc create -f examples/sample-app/docker-registry-config.json
+    F0429 09:22:54.492990   25259 registry.go:154] Docker-registry "docker-registry" does not exist (no service). Pass --create to install.
+
+If it's not running, you can launch it via:
 
-In addition, confirm the IP and Port reported in the services list matches the registry references in your configuration json (e.g. any image tags that contain a registry hostname).  
+    $ openshift admin registry --create --credentials="${OPENSHIFTCONFIG}"
```
Comment (bparees) [inline docs/debugging-openshift.md]: osadm?

Comment (ncdc) [inline docs/debugging-openshift.md]: Fixed

**Comment by ncdc** [comment]
[test]

**Comment by bparees** [comment]
one nit, otherwise lgtm.

**Comment by ncdc** [comment]
nit fixed

**Comment by ncdc** [comment]
Transient failure (UnknownError: Error communicating with the remote browser. It may have died)

[test]

**Comment by bparees** [comment]
[merge]

**Pull request merged**